<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>privatechatwidget</class>
 <widget class="QWidget" name="privatechatwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,1">
   <item>
    <widget class="QLabel" name="friendName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="msgOutput_LW"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="msginput_PTE"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="send_PB">
         <property name="text">
          <string>发送消息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uploadFile_PB">
         <property name="text">
          <string>上传文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
